roles/openshift_version/tasks/first_master_containerized_version.yml

```diff
@@ -6,41 +6,42 @@
     openshift_version: "{{ openshift_image_tag[1:].split('-')[0] if openshift_image_tag != 'latest' else openshift_image_tag }}"
   when:
   - openshift_image_tag is defined
-  - openshift_image_tag != ""
-  - openshift_version is not defined
-  - not (openshift_version_reinit | default(false))
+  - openshift_version is not defined or openshift_version_reinit | bool
 
 - name: Set containerized version to configure if openshift_release specified
   set_fact:
     openshift_version: "{{ openshift_release }}"
   when:
   - openshift_release is defined
-  - openshift_version is not defined
+  - openshift_version is not defined or openshift_version_reinit | bool
```

Comment:
This looks like it will override openshift_version if it was set via openshift_image_tag above.  This is a change in behavior.

I think image_tag is preferable over openshift_release.

Reply:
Probably should add not image_tag defined here as the last condition.